distributions/openhab/src/main/resources/bin/update.ps1

```diff
@@ -1,4 +1,27 @@
-#Requires -Version 5.0
+#Requires -Version 4.0
+function DownloadFiles {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$DownloadSource,
+        [Parameter(Mandatory=$true)]
+        [string]$Outputfile
+    )
+    $client = New-Object System.Net.WebClient
+    try {
+        $client.DownloadFileAsync($DownloadLocation,$Outputfile)
+        while ($client.IsBusy)
+        {
+            Write-Host "Downloading file..."
```

Comment:
The reason I used `Start-BitsTransfer` and `Invoke-WebRequest` (the former didn't work with Cloudbees for some SSL-related reason) is that they handle progress nicely themselves.
Could you maybe integrate https://stackoverflow.com/a/4927295 instead of reporting progress like this?